.github/workflows/test-suite.yml

```diff
@@ -49,6 +49,8 @@ jobs:
       - name: 🧪 Run Web tests
         run: yarn test:web
         working-directory: apps/bare-expo
+        env:
+          EXPO_NO_TYPESCRIPT_SETUP: 1
```

Comment:
the global cli still want to install `@types/react-native`. before upgrade global cli we should keep this.